README.md

```diff
@@ -6,24 +6,28 @@
           Koler
 </h1>
 
+<a href="https://hosted.weblate.org/engage/koler/">
+<img src="https://hosted.weblate.org/widgets/koler/-/svg-badge.svg" alt="Translation startus" />
+</a>
+
 <p align="center">
-         It's a dialer but it's simple and it looks pretty good honestly
+         A dialer, but simple and looks pretty good honestly
```

Comment:
IMO the previous short_description we used is a better one.

`uniquely stylized phone app with customizable features`
(https://github.com/Chooloo/koler/blob/master/fastlane/metadata/android/en-US/short_description.txt)

We do currently using that for F-Droid as well. See [here](https://f-droid.org/packages/com.chooloo.www.koler/)

What do you think @roeiedri & @comradekingu

Reply:
The repeated "it's" has me irked.
Tried shortening it.